src/ioDatas/ReadDatas.java

```diff
@@ -1,5 +1,39 @@
 package ioDatas;
 
+import scheduler.PCB;
+import scheduler.Process;
+import scheduler.State;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class ReadDatas {
 
+    private Log logger;
+
+    public ReadDatas() throws IOException {
+        logger = Log.getInstance();
+    }
+
+    public PCB readFile(int proccessNumber, int processPriority) throws IOException {
+        String processPath = "src/processos/" + (proccessNumber < 10 ? "0" + proccessNumber : proccessNumber) + ".txt";
```

Comment:
If you move the `readQuantum()` and `readPriorities()` methods to this class, you can make `src/processos/` a String variable